experiment_buddy/utils.py

```diff
@@ -67,4 +68,10 @@ async def __remote_time_logger(elapsed: str):
             await response.text()
 
 
+def _get_job_info(jid):
+    result = subprocess.check_output(f"/opt/slurm/bin/sacct --brief -j {jid}".split()).decode()
+    jid, state, exit_code = result.split("\n")[2].split()
+    return state
```

Comment:
Still needed the log in case of return code != 0 https://stackoverflow.com/a/63569771